### ISSUE TITLE
panic importing account from a file

### ISSUE THREAD

**Issue by aricart** [nats-io/nsc#479]
```
~/tmp/X/ngsv2-test-files% less dotnsc-nats-O.attempt-2/accounts/NGS/NGS.jwt
~/tmp/X/ngsv2-test-files% nsc import account --file dotnsc-nats-O.attempt-2/accounts/NGS/NGS.jwt
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x866006]

goroutine 1 [running]:
[github.com/nats-io/nsc/cmd/store.(*Store).IsManaged(..](http://github.com/nats-io/nsc/cmd/store.(*Store).IsManaged(..).)
[github.com/nats-io/nsc/cmd/store/store.go:222](http://github.com/nats-io/nsc/cmd/store/store.go:222)
[github.com/nats-io/nsc/cmd.(*SignerParams).SetDefaults(0xc00014bf80](http://github.com/nats-io/nsc/cmd.(*SignerParams).SetDefaults(0xc00014bf80), 0x70, 0x1, {0xa3bb60, 0xc0001d86c0})
[github.com/nats-io/nsc/cmd/signerparams.go:40](http://github.com/nats-io/nsc/cmd/signerparams.go:40) +0xc6
[github.com/nats-io/nsc/cmd.(*ImportAccount).SetDefaults(0x72edf6](http://github.com/nats-io/nsc/cmd.(*ImportAccount).SetDefaults(0x72edf6), {0xa3bb60, 0xc0001d86c0})
[github.com/nats-io/nsc/cmd/importaccount.go:100](http://github.com/nats-io/nsc/cmd/importaccount.go:100) +0x35
[github.com/nats-io/nsc/cmd.run({0xa3bb60](http://github.com/nats-io/nsc/cmd.run(%7B0xa3bb60), 0xc0001d86c0}, {0x946040, 0xc00014bf80})
[github.com/nats-io/nsc/cmd/action.go:120](http://github.com/nats-io/nsc/cmd/action.go:120) +0x8c
[github.com/nats-io/nsc/cmd.RunMaybeStorelessAction(0xc0001c0280](http://github.com/nats-io/nsc/cmd.RunMaybeStorelessAction(0xc0001c0280), {0xc0000c5640, 0xc000147d80, 0xc000147d58}, {0x946040, 0xc00014bf80})
[github.com/nats-io/nsc/cmd/action.go:95](http://github.com/nats-io/nsc/cmd/action.go:95) +0xf6
[github.com/nats-io/nsc/cmd.createImportAccountCmd.func1(0xc0001c0280](http://github.com/nats-io/nsc/cmd.createImportAccountCmd.func1(0xc0001c0280), {0xc0000c5640, 0x2, 0x2})
[github.com/nats-io/nsc/cmd/importaccount.go:40](http://github.com/nats-io/nsc/cmd/importaccount.go:40) +0x29
[github.com/spf13/cobra.(*Command).execute(0xc0001c0280](http://github.com/spf13/cobra.(*Command).execute(0xc0001c0280), {0xc0000c5600, 0x2, 0x2})
[github.com/spf13/cobra@v0.0.3/command.go:762](http://github.com/spf13/cobra@v0.0.3/command.go:762) +0x60e
[github.com/spf13/cobra.(*Command).ExecuteC(0xd708a0)](http://github.com/spf13/cobra.(*Command).ExecuteC(0xd708a0))
[github.com/spf13/cobra@v0.0.3/command.go:852](http://github.com/spf13/cobra@v0.0.3/command.go:852) +0x2dc
[github.com/spf13/cobra.(*Command).Execute(..](http://github.com/spf13/cobra.(*Command).Execute(..).)
[github.com/spf13/cobra@v0.0.3/command.go:800](http://github.com/spf13/cobra@v0.0.3/command.go:800)
[github.com/nats-io/nsc/cmd.ExecuteWithWriter({0xa2a000](http://github.com/nats-io/nsc/cmd.ExecuteWithWriter(%7B0xa2a000), 0xc0000b2010})
[github.com/nats-io/nsc/cmd/root.go:215](http://github.com/nats-io/nsc/cmd/root.go:215) +0x51
[github.com/nats-io/nsc/cmd.Execute()](http://github.com/nats-io/nsc/cmd.Execute())
[github.com/nats-io/nsc/cmd/root.go:203](http://github.com/nats-io/nsc/cmd/root.go:203) +0x33
main.main()
./main.go:34 +0xaa
```

**Comment by aricart** [comment]
I think this could be due to a decorated JWT, to test, create an account:
```
nsc add account X
nsc describe account --raw -o /tmp/file.jwt
nsc delete account X --force
nsc import account ...
```

**Comment by variadico** [comment]
Hm. I can't seem to reproduce the panic.

This used to fail with an error, not a panic:
```
nsc add account X
nsc describe account --raw -o /tmp/file.jwt
nsc delete account X --force
nsc import account --file /tmp/file.jwt
```
but the import error was fixed in https://github.com/nats-io/nsc/pull/499

I've also tried messing with the output file, but again, I get errors, not panics. 🤔

**Comment by variadico** [comment]
I tried with some jwts like this:
```
-----BEGIN NATS ACCOUNT JWT-----
eyJ0eXAiOiJKV1QiLCJhbGciOiJlZDI1NTE5LW5rZXkifQ.eyJqdGkiOiJNRzNXRTVCMk5ZVTVLREFHTFdYVEZMTFdPSTc1NDZWVUJIRks0WVRTTFVIVktEVVBaREtBIiwiaWF0IjoxNjUzNTAzODM1LCJpc3MiOiJPQlRONE1WSzYzN0dEQllKTUtGMzNYQVQ1QlRaTEJQNk5NNENZVjdJSzVON0VUQ1pHU0cyWDJBNSIsIm5hbWUiOiJYIiwic3ViIjoiQUFVQVpINUgzRlFDQ0NEMjdIWjNMQzJLR1Y0REs0SFJXVVFDV0dZRUtJRDVHRVhZQlJCWlM3WEwiLCJuYXRzIjp7ImxpbWl0cyI6eyJzdWJzIjotMSwiZGF0YSI6LTEsInBheWxvYWQiOi0xLCJpbXBvcnRzIjotMSwiZXhwb3J0cyI6LTEsIndpbGRjYXJkcyI6dHJ1ZSwiY29ubiI6LTEsImxlYWYiOi0xfSwiZGVmYXVsdF9wZXJtaXNzaW9ucyI6eyJwdWIiOnt9LCJzdWIiOnt9fSwidHlwZSI6ImFjY291bnQiLCJ2ZXJzaW9uIjoyfX0._yD9lM5PL7s0MP9udACtMUX3FKsmGqLSWK2Cqm0n8DXqFD2yX3rPOLDbdWwyi9J4TbeKz3nj9on_qZ5qsJisDg
------END NATS ACCOUNT JWT------
```
and
```
************************* IMPORTANT *************************
testing testing
testing testing

-----BEGIN NATS ACCOUNT JWT-----
eyJ0eXAiOiJKV1QiLCJhbGciOiJlZDI1NTE5LW5rZXkifQ.eyJqdGkiOiJNRzNXRTVCMk5ZVTVLREFHTFdYVEZMTFdPSTc1NDZWVUJIRks0WVRTTFVIVktEVVBaREtBIiwiaWF0IjoxNjUzNTAzODM1LCJpc3MiOiJPQlRONE1WSzYzN0dEQllKTUtGMzNYQVQ1QlRaTEJQNk5NNENZVjdJSzVON0VUQ1pHU0cyWDJBNSIsIm5hbWUiOiJYIiwic3ViIjoiQUFVQVpINUgzRlFDQ0NEMjdIWjNMQzJLR1Y0REs0SFJXVVFDV0dZRUtJRDVHRVhZQlJCWlM3WEwiLCJuYXRzIjp7ImxpbWl0cyI6eyJzdWJzIjotMSwiZGF0YSI6LTEsInBheWxvYWQiOi0xLCJpbXBvcnRzIjotMSwiZXhwb3J0cyI6LTEsIndpbGRjYXJkcyI6dHJ1ZSwiY29ubiI6LTEsImxlYWYiOi0xfSwiZGVmYXVsdF9wZXJtaXNzaW9ucyI6eyJwdWIiOnt9LCJzdWIiOnt9fSwidHlwZSI6ImFjY291bnQiLCJ2ZXJzaW9uIjoyfX0._yD9lM5PL7s0MP9udACtMUX3FKsmGqLSWK2Cqm0n8DXqFD2yX3rPOLDbdWwyi9J4TbeKz3nj9on_qZ5qsJisDg
------END NATS ACCOUNT JWT------
```

Seems to be fixed by #499.

Closing for now.
